README.md

```diff
@@ -10,10 +10,8 @@
 - [💻 Getting Started](#getting-started)
   - [Setup](#setup)
   - [Prerequisites](#prerequisites)
-  - [Install](#install)
-  - [Usage](#usage)
-  - [Run tests](#run-tests)
-  - [Deployment](#triangular_flag_on_post-deployment)
+  - [Run test](#test)
```

Comment:
- [ ] It would be nice if you fix this `Run test` link because right now it is broken 😨 Kindly fix it to make your readme descriptive and professional 👍